### PR TITLE
Improve parsing of OpenEats imports

### PR DESCRIPTION
**Pull request by thomaspreece**
Updated the code to be able to parse tags, cuisine and course out of OpenEats. Also parses comments and ratings and allows for transfer of images from OpenEats.

Documentation updated to include process of how to import the extra data

### CODE REVIEW

**Comment by vabene1111** [comment]
Very cool, thank you.

I added a few notes I found while looking at the code, maybe you could take look before I get it merged.

**Comment by vabene1111** [comment]
that is very strange. If you open the files changed tab you dont see any comments anywhere and also not in the timeline on this page? From what I found in GH's docs this should be publicly visible and I also can't find any setting to toggle visibility.

**Review by vabene1111 — CHANGES_REQUESTED** [review]

**Comment by vabene1111** [comment]
ok so i googled again, i needed to submit the review with a change request so that you can see my comments. You learn someting new everyday 😂  now you should be able to see what i wrote

File: docs/features/import_export.md [hunk 1]
```diff
@@ -216,6 +216,8 @@ Store the outputted json string in a `.json` file and simply import it using the
 
 ```
 
+To import your images you'll need to create the folder `openeats-import` in your Tandoor's `recipes` media folder (which is usually found inside `/opt/recipes/mediafiles`). After that you'll need to copy the `/code/site-media/upload` folder from the openeats API docker container to the `openeats` folder you created. You should now have the file path `/opt/recipes/mediafiles/recipes/openeats-import/upload/...` in Tandoor.
```
[the remark this answers is not in the document]
Comment (thomaspreece) [reply docs/features/import_export.md:219]: I think the zip file approach is probably better (as that will allow for processing the images to be web friendly, etc). However, for the time being I think this option is a good stopgap.

File: cookbook/integration/openeats.py [hunk 2]
```diff
@@ -2,24 +2,48 @@
...
         instructions = ''
-        if file["info"] != '':
-            instructions += file["info"]
 
         if file["directions"] != '':
             instructions += file["directions"]
 
         if file["source"] != '':
-            instructions += file["source"]
+            instructions += f'\nRecipe source: [{file["source"]}]({file["source"]})'
```
[the remark this answers is not in the document]
Comment (thomaspreece) [reply cookbook/integration/openeats.py]: I've added `from django.utils.translation import gettext as _` and wrapped my text now but I'm having trouble getting the po files to update. Running `django-admin makemessages -a -s` doesn't seem to update the locale files. I've not used Django before so any tips would be welcome.

[the remark this answers is not in the document]
Comment (vabene1111) [reply cookbook/integration/openeats.py]: Just wrap it, I can recreate the files the nwxt time I update translations

**Pull request merged**